src/Button/ButtonFeatures.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import {EyeIcon, TriangleDownIcon, HeartIcon} from '@primer/octicons-react'
+import React, {useState} from 'react'
+import {Button} from '.'
+
+export default {
+  title: 'Components/Button/Features'
+}
+
+export const Default = () => <Button>Default</Button>
```

Comment:
I'm not using `Template.bind()` for feature stories as an attempt to maintain the story markup for the "Source" addon panel. This makes it easier for someone to copy/paste the example.

I will write an ADR about this format as it impacts our docs work 😄

Reply:
I like this, but I do worry that we're reverting back to `storiesOf` format instead of leveraging CSF correctly in order to make an optional addon functional. If there's overarching appetite for what the addon provides, to the extent that we're willing to exchange for longer, more verbose `*.stories.tsx` files, I guess that's fine. 

Personal take. I like this explicit approach, but I'm a little nervous about future maintainability and interoperability with Storybook direction.

Reply:
An example of the interoperability is Storybook Interactions. I think we'd lose this altogether? 

Example of how it works right now 👇 
https://user-images.githubusercontent.com/13340707/195042883-a409265d-d1f0-4604-8133-bf120beab048.mov

I'll do some testing in Brand to see what happens if we move away from template binding, as there may be a way to do this by attaching the interactions config to the individual Object prototype.

Reply:
Its so frustrating that Storybook doesn't support this addon with the bind syntax _that they recommend_ 😅. I found several open issues on it when I went digging.

What's weird is that the story code shown in Docs does just fine with the template syntax (see below)

![image](https://user-images.githubusercontent.com/18661030/195202260-8ed565f5-f4d5-4833-9211-16dda791b607.png)

The main reason I'm trying to drive adoption of this `Story` source code addon is to support the new Primer docs initiative. We'll be relying on Storybook preview in an iFrame to render code samples, and won't have hand written code samples anymore. Since we're driving people to Storybook, I think its somewhat critical that we provide an easy way to copy code blocks. **If its doable, I would be 100% down to try and build our own addon to accomplish this!** But I would need eng resources for that one as its a bit out of my wheelhouse.